package.json

```diff
@@ -2,8 +2,10 @@
   "name": "create-shared-react-context",
   "version": "1.0.3",
   "description": "A memoized wrapper of React.createContext",
-  "main": "index.js",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
```

Comment:
Would it be worth adding an `exports` field as well, to support the more modern way of describing entrypoints?